frontend/messages/en.json

```diff
@@ -48,6 +51,10 @@
 			"confirmPassword": "Password confirmation",
 			"dob": "Date of birth"
 		},
+		"buttons": {
+			"register": "Register",
+			"login": "Login"
```

Comment:
i know this was already here, but maybe we should stick to our naming convention and use Sign In. Sign up?

Reply:
+1 for "sign up" dominanation over register